### DISCUSSION TITLE
Do we have reference docker-compose file for all RC components setup?

### DISCUSSION THREAD

**Discussion by manojLondhe** [Sunbird-RC/community#660] [Questions (FAQs)] [answered]
Hi,

Do we have reference docker-compose file covering all RC components/services setup including admin UI for quick local dev environment setup?

I could not find it. Pls help.

**Comment by tejash-jl** [comment, marked as answer]
https://github.com/Sunbird-RC/sunbird-rc-core/blob/main/docker-compose.yml

**Reply by manojLondhe** [reply]
thanks!

**Comment by parthlawate** [comment]
@manojLondhe you mentioned you faced a lot of issues due to documentation gaps while installing RC for the first time (Backend and UI).. Could you please help us with 
1. Highlight what areas the documentation is missing in 
2. Contribute those using gitbook/github ? 

cc @Renuka-S @surendrasinghs
